biblatex-iso690.tex

```diff
@@ -374,16 +467,55 @@ \subsubsection{Specific entry types}
 \t|type| field to specify the type of the thesis -- a localization
 term or literal string can be entered. For the list of supported
 localization terms, please refer to section 4.9.2.13 of the
-\t|biblatex| documentation. Names of the supervisor and school
-(institution) can be entered into the fields \t|supervisor| and
-\t|institution|, respectively.
+\t|biblatex| documentation. Available terms are \t|bathesis|,
+\t|mathesis|, \t|phdthesis|, and \t|candthesis|. Names of the
+supervisor and school (institution) can be entered into the fields
+\t|supervisor| and \t|institution|, respectively.
+\newline
+
+\c{Luptak2016thesis}
+\begin{ltxexample}
+@THESIS{Luptak2016thesis,
```

Comment:
I prefer `{...}` instead of `"..."` for field contents. All other examples seem to use `{...}` as well, so I suggest you switch this one to `{...}`.